svpack

```diff
@@ -513,7 +513,7 @@ consequence_parser.set_defaults(func=consequence_main)
 
 tag_zygosity_parser = subparsers.add_parser("tagzygosity", help="Add hetalt and homalt INFO tags that list samples with non-reference genotypes")
 tag_zygosity_parser.add_argument("in_vcf", metavar="IN.vcf")
-tag_zygosity_parser.add_argument("affected", help="Comma-delimited list of affected samples")
+tag_zygosity_parser.add_argument("--affected", help="Comma-delimited list of affected samples")
```

Comment:
I would use a more general parameter name, perhaps `"--samples", help="Comma-delimited list of samples to consider (default: all)`